src/pages/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState } from "react";
+import { useUser } from "../../contexts/UserProvider";
+import { User } from "../../data/models/User";
+
+import EntriesIcon from "../../assets/book.svg";
+import JournalIcon from "../../assets/encrypted.svg";
+import StatsIcon from "../../assets/graph.svg";
+import GroupsIcon from "../../assets/groups.svg";
+import SettingsIcon from "../../assets/settings.svg";
+import LinkButton from "../../components/LinkButton";
+import HappinessForm from "./HappinessForm";
+import { useWindowDimensions } from "../../utils";
+
+export default function Sidebar({ element }: { element: React.ReactElement }) {
+  const { user } = useUser();
+
+  const navConfig = [
+    { title: "Entries", route: "/home", icon: EntriesIcon },
+    { title: "Private Entries", route: "/journal", icon: JournalIcon },
+    { title: "Stats", route: "/statistics", icon: StatsIcon },
+    { title: "Groups", route: "/groups", icon: GroupsIcon },
+    { title: "Settings", route: "/settings", icon: SettingsIcon },
+  ];
```

Comment:
This feels like it should be in its own file since it's a constant that may be used by multiple parts of the app. If you disagree lmk why, otherwise move it to some sort of constants file.

Reply:
Well, we could do that, but it should also include the Profile route, since that's also a valid link (but is on the top of the page). It would make it more difficult to iterate through if I have to deal with a constant component.

I also think that we won't really need all of this information at all times for other components, so I'm not completely sure about making it a constant.